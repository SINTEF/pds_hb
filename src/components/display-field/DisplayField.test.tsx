import React from 'react'
import renderer from 'react-test-renderer'
import { DisplayField } from '.'

it('renders without crashing', () => {
  const tree = renderer
    .create(<DisplayField index="Test" content="TestContent" />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
