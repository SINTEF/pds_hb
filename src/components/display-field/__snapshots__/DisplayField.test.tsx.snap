// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <div
    className="container"
  >
    <label
      className="index"
    >
      Test: 
    </label>
    <label
      className="content"
    >
      TestContent
    </label>
  </div>
  <div>
    <hr
      className="divider"
    />
  </div>
</div>
`;
